components/ui/tabcounter/src/main/java/mozilla/components/ui/tabcounter/TabCounter.kt

```diff
@@ -100,18 +79,14 @@ open class TabCounter @JvmOverloads constructor(
 
         text.setPadding(0, 0, 0, 0)
         text.text = formatForDisplay(count)
-        this.count = count
+        INTERNAL_COUNT = count
     }
 
     private fun tintDrawables(tabCounterTint: Int) {
         val tabCounterBox = DrawableUtils.loadAndTintDrawable(context,
-            R.drawable.mozac_ui_tabcounter_box, tabCounterTint)
+                R.drawable.mozac_ui_tabcounter_box, tabCounterTint)
```

Comment:
We should try to undo these formatting changes since they make the PR a bit noisy when you are trying to review your changes.